### ISSUE TITLE
DataGrid Export menu dropdown not keyboard accessible

### ISSUE THREAD

**Issue by helbnt** [mui/mui-x#5532]
If I navigate to the EXPORT menu item in the toolbar and hit "Enter" on my keyboard, the dropdown menu with the "Export to CSV" option is displayed. I am unable to use the keyboard to select the "Export to CSV" option at this point. The keyboard navigation works as expected COLUMNS, FILTERS, and DENSITY.
